docker-compose.yml

```diff
@@ -42,6 +44,11 @@ services:
       - HTTP_HEADER_AUTH_EMAIL
       - HTTP_HEADER_AUTH_NAME
       - HTTP_HEADER_AUTH_UID
+      - CLOUDFLARE_STORAGE_FEATURE=true
+      - CLOUDFLARE_ACCOUNT_ID
+      - CLOUDFLARE_ACCESS_KEY_ID
+      - CLOUDFLARE_SECRET_ACCESS_KEY
+      - CLOUDFLARE_BUCKET
```

Comment:
It makes sense to include all of these so that docker passes them through, but let’s not set any defaults here. And let’s keep the default as postgres, not cloudflare. And in terms of access these config ENV, we should do that through `Feature.` and `Setting.`. If you search the codebase for “google_auth” (not case sensitive) you can see a feature which has both Feature and Setting.